lib/engines/hcktest/tests.rb

```diff
@@ -132,11 +132,37 @@ def wait_queued_test(id)
       end
     end
 
-    def test_parameters(test_name)
+    def read_test_config(test_name, config)
```

Comment:
"Read" sounds like it's reading a file. An alternative is "select", which is the word used in `Array#select` called in this method.